ansible/roles/schulcloud-client-core/templates/client-configmap-files.yml.j2

```diff
@@ -50,7 +51,7 @@ data:
             use * as value for
             defaultSrc, fontSrc, styleSrc, scriptSrc, imageSrc,
             connectSrc, mediaSrc, objectSrc, prefetchSrc, childSrc,
-            frameSrc, workerSrc, frameancestorsSrc, formactionSrc, baseuriSrc
+            frameSrc, workerSrc, frameAncestors, formactionSrc, baseuriSrc
```

Comment:
same question here, if we need to update https://acc.bettermarks.com reference below

Reply:
no